src/show/pages/private/EditProject/index.js

```diff
@@ -3,7 +3,7 @@ import StepWizard from 'react-step-wizard';
 import { useParams } from 'react-router-dom';
```

Comment:
By looking at this file I can notice editing a project is not working anymore since you are sending `selectedProject` and `setProject`, but in the actual steps you are not using them because you are only using the transient state and transient action. So sending those props are redundant. Delete them and try to initialise the transient state when the component mounts with the fetched project data.

Reply:
done